packages/form-utils/src/lib/Switch.stories.tsx

```diff
@@ -29,34 +31,94 @@ const meta: Meta<typeof Switch> = {
 export default meta;
 
 export const _Switch: StoryObj<typeof Switch> = {
-  render: (args: SwitchProps) => <Switch inputProps={{ 'aria-label': 'example' }} {...args} />,
+  render: (args: SwitchProps) => {
+    const [checked, setChecked] = useState(false);
+    return (
+      <FormControlLabel
+        control={<Switch {...args} onChange={(event) => setChecked(event.target.checked)} />}
+        label={checked ? 'On' : 'Off'}
+      />
+    );
+  },
+  args: {
+    sx: { marginRight: '6px' },
+    inputProps: {
+      'aria-label': 'example',
```

Comment:
Personally, I prefer the bare component with no label for the primary story, but if we're keeping `FormControlLabel` let's ditch the `aria-label`

Reply:
I had a conversation with UX about this. The pattern in zeroheight expects an action label, so this would be the most common use case.

Reply:
That makes sense. I switched back to the Switch story being a bare Switch component and added the With Label story.